cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -828,8 +828,20 @@ def set_args(self, request, verbose=False):
         else:
             raise NotImplementedError
 
-        self.par_dic = args.to_dict()
-
+        self.par_dic = {}
+        for k, v in args.to_dict().items():
+            if k in ['catalog_selected_objects', 'selected_catalog']:
```

Comment:
this will change again in the future with #551

Reply:
True, it may change. But it's not always related with file upload. The catalogue may also be added as JSON-encoded string (this happens if the user selects some lines in the frontend "catalog" window).
I decided to completely mask these parameters so that changes are minimal and garanteed to be backward-compatible